tests/suite.bats

```diff
@@ -673,6 +673,7 @@ EOF
   assert_has_line "${STACKER_NAMESPACE}-dependent-rollback-parent: submitted (rolling back new stack)"
   assert_has_line "${STACKER_NAMESPACE}-dependent-rollback-parent: failed (rolled back new stack)"
   assert_has_line "${STACKER_NAMESPACE}-dependent-rollback-child: failed (dependency has failed)"
+  assert_has_line "The following stacks failed: dependent-rollback-parent, dependent-rollback-child"
```

Comment:
This is actually just a revert to how 1.1 behaves: https://github.com/remind101/stacker/blob/1.1.4/tests/suite.bats#L637